packages/browser-wallet/src/popup/pages/ChangePasscode/i18n/en.ts

```diff
@@ -0,0 +1,11 @@
+const t = {
+    changePasscode: 'Change passcode',
+    incorrectPasscode: 'Incorrect passcode',
+    labels: {
+        currentPasscode: 'Enter current passcode',
+        newPasscode: 'Enter new passcode',
+        newPasscodeRepeated: 'Enter new passcode again',
```

Comment:
```suggestion
        newPasscodeRepeated: 'Confirm new passcode',
```
Do you not agree with this linguistic improvement?